### ISSUE TITLE
Enhancement: More Detailed Party Sharing

### ISSUE THREAD

**Issue by BenisonQPS** [JamesPeters98/ChestsPlusPlus#24]
It's fantastic that you can deny/share to select members or all players. Sometimes however it would be nice to only allow certain chest to a party member. eg. as a team you create am Iron Farm, share the [ChestLink], [IronFarm] chest. You build a Gold farm with someone else, share that chestlink chest. As it stands currently, both members would have access to both Iron AND Gold chests. Not exactly sure how hard this would be to implement. Keep up the good work!

**Comment by JamesPeters98** [comment]
Yes, this is something I have considered, I will probably look into this functionality in the party menu soon.

However for now you can invite members to individual chests or set a chest to public using the following commands:

```
/chestlink member [add/remove ] or [list ] "Add, remove or list members of a group"
/chestlink member [add-to-all/remove-from-all] "Add/Remove a player to all of your ChestLinks"
/chestlink setpublic <true/false> "Set a ChestLink to be accessible by anyone."
```
